jira/client.py

```diff
@@ -1739,12 +1741,20 @@ def createmeta_issuetypes(
                 "Use 'createmeta' instead."
             )
 
-        return self._get_json(f"issue/createmeta/{projectIdOrKey}/issuetypes")
+        return self._get_json(
+            f"issue/createmeta/{projectIdOrKey}/issuetypes",
+            params={
+                "startAt": startAt,
+                "maxResults": maxResults,
+            },
+        )
 
     def createmeta_fieldtypes(
         self,
         projectIdOrKey: str | int,
         issueTypeId: str | int,
+        startAt: int = 0,
+        maxResults: int = 50,
```

Comment:
Can you update Docstring for new arguments.

Reply:
@nickmitricka Thanks for the review! I've updated doc strings for the both methods.